models/renalStructures_UNEST_segmentation/docs/README.md

```diff
@@ -1,7 +1,13 @@
 # Description
 A pre-trained model for inferencing volumetric (3D) kidney substructures segmentation from contrast-enhanced CT images (Arterial/Portal Venous Phase).
+<<<<<<< HEAD
 A tutorial and release of model for kidney cortex, medulla and collecting system segmentation.
 
+=======
+
+A tutorial and release of model for kidney cortex, medulla and collecting system segmentation.
+
+>>>>>>> db7fd87494d8c7ad9b766bdf6e4010c57a205c79
```

Comment:
Please resolve the conflicts here, only keep the right doc.

Thanks.

Reply:
Deleted, thanks!